src/Patch/Cli/Command/ApplyCommand.php

```diff
@@ -114,10 +114,26 @@ protected function configure()
      */
     public function setPhpVersion($version)
     {
+        // Check if the current version has metadata in it
+        // example : 5.6.0-1ubuntu3.25
+        if (count(explode('-', $version)) > 1) {
+            $versionWithMetadata = explode('-', $version);
```

Comment:
Minor optimisation might be to explode once and set in a variable then use that variable here.